### PR TITLE
Ft responsive designs

### PR DESCRIPTION
**Pull request by Gabkings**
#### What does this PR do?
This PR demonstrates the mastering of developing a responsive website by discovering browser breaking points.
#### Description of Task to be completed?

- The HTML mark up a layout using the required tags for headers, main and footer

- Adding styling to HTML sections in the index page

- Identifying browser breaking points for responsive 

- Using of media queries for designing the breaking points styles for different devices

#### How should this be manually tested?

1. Clone the project via this link: [Project](https://github.com/Gabkings/Building-with-Responsive-Design.git)
2 `cd Building-with-Responsive-Design`
3 Open the index.html file in your favorite browser

#### Any background context you want to provide?
CSS and HTML are required to understand the project structures and styling
#### What are the relevant pivotal tracker stories?
N/A
#### Questions:
All the questions are welcome

### CODE REVIEW

**Review by andresporras3423 — COMMENTED** [review]
### Status: Invalid submission :warning:
**Hello Gitonga Gitonga** 
Friendly reminder: You should make sure that your repo looks professional before review. Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/. Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO @Gabkings AND @andresporras3423  GREAT PROGRESS SO FAR. :clap:  AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT.

THERE ARE SOME IMPROVEMENTS TO BE DONE KINDLY FIX THE FOLLOWING :arrow_down: 

- [ ] Kindly add a linter to your project. Remember this is a requirement for your project to be considered complete.

- [ ] There are some spelling errors in your commit messages try and avoid this kindly. Remember they should be descriptive.

descriptive_commits

- [ ] Kindly add images to display your project on the readme file. Also, add a live preview of the project.

- [ ] Kindly if you check the menubar you will realize it doesn't look like the original. You need to fix this.
menu bar

- [ ] There is an overflow on the x-axis. Kindly you need to fix this. Try to align the elements well in your webpage. This will fix that issue
overflow

- [ ] At your latest section. Your elements are aligned vertically which is at the desktop view. This should not be the case. kindly check below :arrow_down: 

### CLONE
latest-sec clone
### ORIGINAL
latest-sec original

- [ ] At mobile view this section is losing responsiveness kindly fix it
MOB

- [ ] Also there is an overflow at the x-axis at the mobile view. Kindly you need to fix this

### if you have any question reach me via slack on @peterrobert

### HAPPY CODING :computer:

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  
#### Hey team,  your project is looking really good, however, to accomplish as the actual page, you required to do these changes, listed below:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
In large, medium and small screens your footer have a font-color that is terrible to read. Fix that
- [ ] In large, medium and small screens, your page faces overlapping in a some sections, check my images and kindly improve that.
- [ ] In large, medium and small screens, your page faces list style in some elements that should not have list style, check my images and change that
- [ ] In large, medium and small screens, your page have some margin that shouldn't exist in the footer and at the navbar, Take care of that
- [ ] In large, medium and small screens, your page faces some horizontal overflow, check my images and fix that.
- [ ] In large, medium and small screens, your nav bar don't looks like the original one, check those images below and please improve it


#### Large & Medium screen issues:
<table>
 <tr>
 <th>Yours</th>
 <th>Original</th>
 </tr>
 <tr>
 <td> Your navbar don't look like the original one, is overlapping the image and have borders when it shouldn't and you're facing overflow at the x-axis</td>
 <td></td>
 </tr>
 <tr>
 <td>You have style issues with those list style and overflow at those images on latest funding rounds</td>
 <td></td>
 </tr>
  <tr>
 <td>You have style issues with these text at those ribbons, and you're missing one image</td>
 <td></td>
 </tr>
  <tr>
 <td>You have style issues with the color of that text (we barely can read it),  you have list style where it shouldn't and have borders at the footer, remove them.</td>
 <td></td>
 </tr>
</table>

#### Small screen issues:
<table>
 <tr>
 <th>Yours</th>
 <th>Original</th>
 </tr>
 <tr>
 <td></td>
 <td></td>
 </tr>
 <tr>
 <td></td>
 <td></td>
 </tr>
 <tr>
 <td>
Add that missing image, fix that overlapping, change that text-color, put all those icons in only one line like the original one </td>
 <td></td>
 </tr>
</table>

## [OPTIONAL CHANGES]
- [ ] When making comments, remember to just make relevant comments about the code, is a good practice to avoid obvious comments. 💪
- [ ] Add a favicon to make it more similar to the original page;

⚠ These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:
- [ ] Kindly remember to make a professional and descriptive README.md (with live link, screenshot, and brief description of the project) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/))

- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image




## Summary
Just have to implement that minor changes listed above, and I know that It will be easy for you! 🔧



## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  
#### Hey @Gabkings I didn't see any changes, so my review is still the same previously required changes, please implement all the REQUIRED changes in order to have your project approved:
#### Your project is looking good, however, to accomplish as the actual page, you required to do these changes, listed below:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
In large, medium and small screens your footer have a font-color that is terrible to read. Fix that
- [ ] In large, medium and small screens, your page faces overlapping in a some sections, check my images and kindly improve that.
- [ ] In large, medium and small screens, your page faces list style in some elements that should not have list style, check my images and change that
- [ ] In large, medium and small screens, your page have some margin that shouldn't exist in the footer and at the navbar, Take care of that
- [ ] In large, medium and small screens, your page faces some horizontal overflow, check my images and fix that.
- [ ] In large, medium and small screens, your nav bar don't looks like the original one, check those images below and please improve it


#### Large & Medium screen issues:
<table>
 <tr>
 <th>Yours</th>
 <th>Original</th>
 </tr>
 <tr>
 <td> 

- Your navbar don't look like the original one, is overlapping the image and have borders when it shouldn't and you're facing overflow at the x-axis</td>
 <td></td>
 </tr>
 <tr>
 <td>


- You have style issues with those list style and overflow at those images on latest funding rounds</td>
 <td></td>
 </tr>
  <tr>
 <td>


- You have style issues with these text at those ribbons, and you're missing one image</td>
 <td></td>
 </tr>
  <tr>
 <td>


- You have style issues with the color of that text (we barely can read it),  you have list style where it shouldn't and have borders at the footer, remove them.</td>
 <td></td>
 </tr>
</table>

#### Small screen issues:
<table>
 <tr>
 <th>Yours</th>
 <th>Original</th>
 </tr>
 <tr>
 <td></td>
 <td></td>
 </tr>
 <tr>
 <td></td>
 <td></td>
 </tr>
 <tr>
 <td>


- Add that missing image, fix that overlapping, change that text-color, put all those icons in only one line like the original one </td>
 <td></td>
 </tr>
</table>

## [OPTIONAL CHANGES]
- [ ] Add a favicon to make it more similar to the original page;

⚠ These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:
- [ ] Kindly remember to make a professional and descriptive README.md (with live link, screenshot, and brief description of the project) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/))

- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image




## Summary
Just have to implement that minor changes listed above, and I know that It will be easy for you! 🔧



## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: index.html [hunk 1]
```diff
@@ -0,0 +1,859 @@
...
+                        <li><a href="#">PLUGGED</a></li>
+                        <li><a href="#">FUNDAMENTALS</a></li>
+                        <li><a href="#">WORK 2030</a></li>
+                    </ul>
+                    <button>
+						<i class="search-icon"></i>
+					</button>
+                </div>
+            </div>
+        </nav>
```
Comment (SpaYco) [inline index.html]: - [ ] your navbar is missing the yellow part

- [ ] you navbar should be on the very top with no spaces on top or on the sides on all sizes

| Yours | Original |
| --- | --- |
| image | image |

- [ ] you need to have a hamburger button on the small screen size

| Yours | Original |
| --- | --- |
| image | image |

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] Your page has a horizontal overflow on all screen sizes, please fix that! 
_Pro Tip: give everything a border to find the element causing the problem_

image

- [ ] you have double dots on the left of the headlines _[Learn More](https://www.bitdegree.org/learn/css-list-style)_

image

- **I Attached More Changes Below 👇**

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,1023 @@
...
+}
+
+.author a:hover {
+  color: white;
+}
+
+.topNav li a:hover,
+.top-footer li a:hover {
+  color: white;
+}
```
Comment (andresporras3423) [inline css/styles.css]: Group selectors with the same style definition

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,1023 @@
...
+.img-news {
+  width: 100%;
+  height: 135px;
+  margin-bottom: 4px;
+}
+
+/* .timer {
+  padding-top: 4px;
+  color: #aaa;
+} */
```
Comment (andresporras3423) [inline css/styles.css]: Remove commented code

File: index.html [hunk 4]
```diff
@@ -0,0 +1,864 @@
...
+                        <button>
+                            <i class="search-icon"></i>
+                        </button>
+                        <button class="humbg">
+                            <i class="fa fa-bars" aria-hidden="true"></i>
+                        </button>
+                </div>
+            </div>
+            <a href="#">
+                <div class="nav-message nav-message-announcement" style="pointer-events:none">
```
Comment (andresporras3423) [inline index.html]: Remove inline style

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Gitonga Gitonga** 
Good job with your new submission for this project :muscle:  :muscle:
Your page follows the layout of the original  :clap: :clap: :clap:
There are still some improvements to do before you can go ahead:
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.
- [ ] Remove inline styles in your HTML file
- [ ] Group selectors
- [ ] In the small screen size, check out the navbar, math with the original. Besides, remove the double bullet point in the categories section:

| Yours | Original |
|:---:|:---:|
|image | image |

- [ ]  In the small screen size, check the footer of your page, match with the original:

| Yours | Original |
|:---:|:---:|
|image | image |

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: index.html (hunk 4 above)
Comment (MuriloRoque) [reply index.html]: - [ ] You still didn't fix this inline styling, please remove it.

File: css/styles.css (hunk 2 above)
Comment (MuriloRoque) [reply css/styles.css]: - [ ] These selectors are still not grouped, please group them.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested 🔴 

### Hello @Gabkings 👋 

### Thank you for your hard work, but it looks like the issues from the last TSE are still happening, except for the commented code, please check my comments.

- [ ] In the small screen size, check out the navbar, match with the original. Besides, remove the double bullet point in the categories section:

***Yours:***
image
<br>
***Original:***
image
<br>

- [ ] In the small screen size, check the footer of your page, match with the original:

***Yours:***
image
<br>
***Original:***
image
<br>

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow

**Comment by Gabkings** [comment]
I added the changes you have requested

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 

You Still haven't changed the navbar issues requested by both previous TSEs, you must fix them in order for this project to be approved

## Required Changes ♻️ 

# these changes below are mandatory in order for the project to be accepted

- [ ] Your navbar should take the full width of the screen on all sizes, yours have a gap on the sides and on the top section, remove those gaps or try using `position: fixed;`

| Yours | Original |
| --- | --- |
| image | image |

Quote from the previous TSE reply: 

> ###
> * [ ]  In the small screen size, check out the navbar, match with the original. Besides, remove the double bullet point in the categories section:
> 
> _**Yours:**_
> image
> 
> 
> _**Original:**_
> image
> 
> 
> * [ ]  In the small screen size, check the footer of your page, match with the original:
> 
> _**Yours:**_
> image
> 
> 
> _**Original:**_
> image


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by SpaYco** [comment]
P.S: if you are stuck/have a problem solving this issue, [contact a Full-Time TSE](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-)

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Well done guys you are almost there :checkered_flag:, just some minor changes :hammer: :wrench: :toolbox:  and you are done.

- [ ] Fix the header on tablet view:
image

- [ ] the yellow link should not be transparent:
image

- [ ] Fix the articles on your page by adding this code to there ul selector: 
    list-style: none;
    padding: 0;
to remove the list style and the unnecessary padding, from the latest news and latest deals sections:
image

- [ ] same for this section to align the title with the article images:
image

### Please make all the required changes before you submit another code review
## Happy Coding :tophat:

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved! 

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**